ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
+\section{State of the Art}
+There are criteria that make a tool better for modeling a NRHES. In order to select a tool to model a NRHES, it is necessary to understand what characteristics a model must possess in order to provide accurate information. For example, as discussed below, it is important for a NRHES model to incorporate the dynamic nature of the system. Understanding the differences among the underlying mathematical models as well as the software functionality is essential in determining whether and how a nuclear fuel cycle simulator (NFCS) will be useful to NRHES modeling.
+As mentioned above, the model currently under development by the Oak Ridge National Laboratory, the Argonne National Laboratory, and the Idaho National Laboratory has reviewed the essential characteristics for a computational model of a NRHES. Rabiti et al. (2015) discusses the important elements of a NRHES computational model in detail, describing the basic requirements of the software as a "computational representation of the thermal, mechanical, chemical, and electrical processes in the systems as process units, reactors, manufacturing plants, and energy delivery (to the appropriate point of interface with the market transaction, such as an electricity bus, or a product depot or distribution terminal where a commodity price is established)"\cite{Rabiti2015}. The model in development by the national laboratories combines RAVEN (Reactor Analysis and Virtual Control Environment) for the stochastic aspects of the system and Modelica for modeling the various components in the NRHES.
 
```

Comment:
Updated section to State of the art.  Moved dynamic modeling discussion to II.B. under Distinct Capabilities